packages/cryptography/src/Mnemonic.js

```diff
@@ -125,7 +146,77 @@ export default class Mnemonic {
             return this.toLegacyPrivateKey();
         }
 
-        return this._toPrivateKey(passphrase);
+        let { keyData, chainCode } = await this._toKeyData(
+            passphrase,
+            ED25519_SEED_TEXT
+        );
+
+        for (const index of path) {
+            ({ keyData, chainCode } = await slip10.derive(
+                keyData,
+                chainCode,
+                index
+            ));
+        }
+
+        const keyPair = nacl.sign.keyPair.fromSeed(keyData);
+
+        if (CACHE.privateKeyConstructor == null) {
+            throw new Error("PrivateKey not found in cache");
+        }
+
+        return CACHE.privateKeyConstructor(
+            new Ed25519PrivateKey(keyPair, chainCode)
+        );
+    }
+
+    /**
+     * Recover an ECDSA private key from this mnemonic phrase, with an
+     * optional passphrase.
+     *
+     * @param {string} [passphrase]
+     * @param {number[]} [path]
+     * @returns {Promise<PrivateKey>}
+     */
+    async toEcdsaPrivateKey(passphrase = "", path = ECDSA_ETH_PATH) {
```

Comment:
I think we've settled that all Hedera keys should use the `3030'` path, ED and EC.  So this default should change.
However, we can used the Eth path to validate key derivations to prove the framework is working properly.

Reply:
Honestly, not following with the last statement there.

Reply:
Similar to my statement.
Basically even though we use the default Hedera one we should have a test that sets the path as the Ethereum path for coverage